lib/spack/spack/test/data/config/concretizer.yaml

```diff
@@ -5,3 +5,5 @@ concretizer:
     host_compatible: false
   duplicates:
     strategy: minimal
+  concretization_cache:
```

Comment:
Is this the right place to set this default for the tests? Some of the test failures looked like they were trying to use the conc cache...